### DISCUSSION TITLE
Hermes 3 405B API access is free for this month - I don't suppose this can be quickly added?

### DISCUSSION THREAD

**Discussion by catbalony** [SakanaAI/AI-Scientist#59] [Ideas]
TLDR - I don't know how to code. I tried to add this functionality, but it's beyond my ability to do with llm's. 

https://openrouter.ai/models/nousresearch/hermes-3-llama-3.1-405b:extended
https://openrouter.ai/models/nousresearch/hermes-3-llama-3.1-405b

API access is free at Open Router as a promo for the remaining time. I wanted to quickly get some free projects running to test this out. 

Thanks guys.

**Comment by BradKML** [comment]
As long as it is open to being downloaded in platforms like Ollama, self-hosting could be an alternative?

**Comment by aliomri7373-ui** [comment]
سلام
